tools/loadResources

```diff
@@ -72,12 +72,38 @@ else
 fi
 echo "Determined the json contains resources of type $resType"
 
+
+
 # Loop thru each resource and publish it
 keys=$(jq -r 'to_entries[] | .key' <<< $json)
 checkexitcode $? 'getting keys from the specified json'
 for k in $keys; do
     name=${k#*/}    # strip the org from the beginning of the key
     resourceJson=$(jq -c ".[\"$k\"] | del(.owner) | del(.lastUpdated) | .public = false" <<< $json)
+
+    OLD_HZN_ORG_ID=${k%%/*}
```

Comment:
Name this like a regular variable (e.g. oldOrgId) so it doesn't get mistaken for an environment variable